### PR TITLE
New netlink Interface methods

### PR DESCRIPTION
**Pull request by tpantelis**
...in support of https://github.com/submariner-io/submariner/pull/1716 as outlined in review comments. See individual commits for details.

### CODE REVIEW

**Review by skitt — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Comment by maryamtahhan** [comment]
It looks like this is good to go, is there a reason for not merging?

**Comment by tpantelis** [comment]
> It looks like this is good to go, is there a reason for not merging?

I was waiting for your approval 😄

**Pull request merged**
